hybra/wordcloud/module_wordclouds.py

```diff
@@ -0,0 +1,50 @@
+import os
+import time
+from string import Template
+import random
+
+from IPython.core.display import HTML, display
+
+from collections import Counter
+
+path = os.path.dirname(os.path.abspath(__file__))
+
+def create_wordcloud( data, plt, stopwords = ["the", "a", "or", "tai", "and", "ja", "to", "on", "in", "of", "for", "is", "i", "this", "http", "www", "fi", "com"], width=850, height=350 ):
+
+    import codecs
+    html_template = Template( codecs.open( path + '/wordcloud.html', 'r').read() )
+    js_template = Template(codecs.open(path + '/wordcloud.js', 'r').read())
+    css_text = codecs.open(path + '/wordcloud.css', 'r').read()
+
+    texts = ""
+    for node in data:
+        text = node['text_content']
+        text = text.replace(",", "")
+        text = text.replace(".", "")
+        text = text.replace("!", "")
+        text = text.replace("?", "")
+        for word in text.split(" "):
+            if word not in stopwords:
+                texts += word + " "
+
+    frequencies = Counter(texts.split())
+    freqs_list = []
+    colors = ["#A5E6BA", "#9AC6C5", "#7785AC", "#248757", "#360568", "#F0544F", "#e07f9f", "#1d7059", "#3e6282"]
+
+    for key, value in frequencies.iteritems():
+        freqs_list.append({"text":encode_utf8(key),"size":str(value), "color": random.choice(colors)})
+
+    graph_div_id = int(time.time() * 1000)
```

Comment:
mahdollisesti clashaa, satunnaiset on välillä vähän hassuja. voisiko tätä välttää?

Reply:
Tämä ei .ole satunnainen, time.time() palauttaa "Return the time in seconds since the epoch as a floating point number" eli tietääkseni tuo on uniikki (kopioin tuon lähestymistavan timelinesta)

Reply:
my mistake.